### DISCUSSION TITLE
GH Projects Beta created on org but not accessible to collaborators

### DISCUSSION THREAD

**Discussion by andre-wells** [community/community#6805] [Projects and Issues] [answered]
Hi all

My personal account has been enrolled and has access to the the new Projects Beta.  I'm able to create these new Projects in my own profile.

In one of my organizations, I was unable to create a new project but the org owner could.  However, even when I was added as an admin collaborator, I can't see the new project (and a direct link gives me a 404).  Any ideas?

**Org owners Projects view (I don't see "Projects (Beta) in my view)**

![image](https://user-images.githubusercontent.com/66315105/139038692-fbb58766-9e54-4ede-8ceb-515462137718.png)

**Access list for Project (both Org owner and I are admin).**

![image](https://user-images.githubusercontent.com/66315105/139038803-67130499-4937-4f08-8b35-3fc114828fa1.png)

**Comment by timoyates** [comment, marked as answer]
@andre-wells Thanks for getting in touch. We're rolling out some changes today that might address the issue. Would you mind trying again and letting us know if not? Cheers!

**Reply by andre-wells** [reply]
Looks like Projects (beta) is back on the menu!
